ports/llfio/portfile.cmake

```
@@ -0,0 +1,78 @@
+message(WARNING [=[
+LLFIO depends on Outcome which depends on QuickCppLib which uses the vcpkg versions of gsl-lite and byte-lite, rather than the versions tested by QuickCppLib's, Outcome's and LLFIO's CI. It is not guaranteed to work with other versions, with failures experienced in the past up-to-and-including runtime crashes. See the warning message from QuickCppLib for how you can pin the versions of those dependencies in your manifest file to those with which QuickCppLib was tested. Do not report issues to upstream without first pinning the versions as QuickCppLib was tested against.
+]=])
+
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ned14/llfio
+    REF 9294f05d48220a6a5d6d307da5fe8f236dd020cf
+    SHA512 00bd49fe568ba2a840917de8d5b8d398d7480b2431f99f8c3b282fc60785d8664869fd83e8ac56ffc9a1eaa0b991c5d8a97ab5e53e20e5567f3d75577ad4feb4
+    HEAD_REF develop
+)
+
+if(WIN32)
```

```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
```
WIN32 here refers to the host machine, not the target